docs/index.restdown

```diff
@@ -143,6 +143,7 @@ below (and `listen()` takes the same arguments as node's
 ||certificate||String||If you want to create an HTTPS server, pass in the path to PEM-encoded certificate and key||
 ||key||String||If you want to create an HTTPS server, pass in the path to PEM-encoded certificate and key||
 ||formatters||Object||Custom response formatters for `res.send()`||
+||ignoreUncaughtExceptions||Boolean||When true (default is false), turns off restify's uncaughtException handler||
```

Comment:
Can we reverse this so the default is to turn it off? I'd rather restify ship out of the box to _not_ swallow errors.